pkg/plugins/platform/local.go

```diff
@@ -174,17 +161,17 @@ func (l *LocalApp) ImportDB(imPath string, extPath string) error {
 		if extPath == "" {
 			extPathPrompt = true
 		}
-		fmt.Println("Provide the path to the database you wish to import.")
-		fmt.Println("Import path: ")
+		util.UserOut.Println("Provide the path to the database you wish to import.")
+		fmt.Print("Import path: ")
 
 		imPath = util.GetInput("")
 	}
 
 	importPath, err := appimport.ValidateAsset(imPath, "db")
 	if err != nil {
 		if err.Error() == "is archive" && extPathPrompt {
-			fmt.Println("You provided an archive. Do you want to extract from a specific path in your archive? You may leave this blank if you wish to use the full archive contents")
-			fmt.Println("Archive extraction path:")
+			util.UserOut.Println("You provided an archive. Do you want to extract from a specific path in your archive? You may leave this blank if you wish to use the full archive contents")
+			fmt.Print("Archive extraction path:")
```

Comment:
This should be `util.UserOut`, right?

Reply:
No, same reason, apparently no way to get Logrus to not emit a CR, so prompting looks weird.